troubleshooting/known_issues/SBPS_boot_fail.md

```diff
@@ -0,0 +1,114 @@
+# Boot content projection to Compute/UANs via iSCSI fails if the image does not have a `etag`

```

Comment:
```suggestion
# Boot content projection services can fail if an image does not have a `etag`
```

Reply:
Just to be clear the image being booted may not necessarily be the image that is causing the issue.   It could be any image.